README.md

```diff
@@ -29,3 +29,128 @@ make init
+
+Verify that your node has finished synching and it is caught up with the network
+
+`curl -so- http://localhost:26657/status | jq .result.sync_info.catching_up`
+Wait until you see the output: "false"
+
+3. Fund account.
+
+`l1_node.sh` script generates keys, you can find created account information in `data/*.account_info`. Get the address from the file and fund, on testnets you can use faucet `https://faucet.${NETWORK}.allora.network`.
+
+4. Stake validator
+
+Here's an example with Values which starts with a stake of 10000000uallo.
+
+All the following command needs to be executed inside the validator container.
+Run `docker compose exec validator0 bash` to get shell of the validator.
+
+You can change `--moniker=...` with a human readable name you choose for your validator.
+and `--from=` - is the account name in the keyring, you can list all availble keys with `allorad --home=$APP_HOME keys --keyring-backend=test list`
+
+Create stake info file:
+```bash
+cat > stake-validator.json << EOF
+{
+    "pubkey": $(allorad --home=$APP_HOME comet show-validator),
+    "amount": "1000000uallo",
+    "moniker": "myvalidator",
+    "commission-rate": "0.1",
+    "commission-max-rate": "0.2",
+    "commission-max-change-rate": "0.01",
+    "min-self-delegation": "1"
+}
+EOF
+```
+
+Stake the validator
+```bash
+allorad tx staking create-validator ./stake-validator.json \
+    --chain-id=edgenet \
+    --home="$APP_HOME" \
+    --keyring-backend=test \
+    --from=validator0
+```
+The command will output tx hash, you can check its status in the explorer: `https://explorer.edgenet.allora.network:8443/allora-edgenet/tx/$TX_HASH`
+
+
+5. Verify validator setup
+
+### Check that the validator node is registered and staked
+
+```bash
+VAL_PUBKEY=$(allorad --home=$APP_HOME comet show-validator | jq -r .key)
+allorad --home=$APP_HOME q staking validators -o=json | \
+    jq '.validators[] | select(.consensus_pubkey.value=="'$VAL_PUBKEY'")'
+```
+
+- this command should return you all the information about the validator. Similar to the following:
+```
+{
+  "operator_address": "allovaloper1n8t4ffvwstysveuf3ccx9jqf3c6y7kte48qcxm",
+  "consensus_pubkey": {
+    "type": "tendermint/PubKeyEd25519",
+    "value": "gOl6fwPc19BtkmiOGjjharfe6eyniaxdkfyqiko3/cQ="
+  },
+  "status": 3,
+  "tokens": "1000000",
+  "delegator_shares": "1000000000000000000000000",
+  "description": {
+    "moniker": "val2"
+  },
+  "unbonding_time": "1970-01-01T00:00:00Z",
+  "commission": {
+    "commission_rates": {
+      "rate": "100000000000000000",
+      "max_rate": "200000000000000000",
+      "max_change_rate": "10000000000000000"
+    },
+    "update_time": "2024-02-26T22:50:31.187119394Z"
+  },
+  "min_self_delegation": "1"
+}
+```
+### Check the voting power of your validator node
+*NOTE:* please allow 30-60 seconds for the output to be updated
+
+`allorad --home=$APP_HOME status | jq -r '.validator_info.voting_power'`
+- Output should be > 0
```

Comment:
Not sure if this is needed rn since we don't have a gov module yet

Reply:
thios is a readonly, just to check if it staked successfully

and it works, so why not)))